From: Jakub Hrozek &lt;jhrozek@redhat.com&gt;
Date: Mon, 3 Jun 2019 13:30:27 +0200
Subject: [PATCH] tests: Remove the use of an expired cert in tests as a
workaround

The test used to verify the signature of the IDP metadata in
test13_test_lasso_server_load_metadata() expired at &#34;Mar 23 09:51:37
2019 GMT&#34;

This patch just removes the certificate usage from
lasso_server_load_metadata() which means we don&#39;t validate the metadata,
but we can keep the rest of the test at least.
---
 tests/basic_tests.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/basic_tests.c b/tests/basic_tests.c
index 398d789..bbb0972 100644
--- a/tests/basic_tests.c
+++ b/tests/basic_tests.c
@@ -1983,7 +1983,7 @@ START_TEST(test13_test_lasso_server_load_metadata)
 	block_lasso_logs;
 	check_good_rc(lasso_server_load_metadata(server, LASSO_PROVIDER_ROLE_IDP,
 				TESTSDATADIR "/metadata/renater-metadata.xml",
-				TESTSDATADIR "/metadata/metadata-federation-renater.crt",
+				NULL,
 				&blacklisted_1, &loaded_entity_ids,
 				LASSO_SERVER_LOAD_METADATA_FLAG_DEFAULT));
 	unblock_lasso_logs;
